.github/workflows/ci.yml

```diff
@@ -4,6 +4,10 @@
 name: Admin Components React CI
 
 on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
There is no `master` branch, it is `main` on this repo.

Reply:
switched